Unfortunately our CI logs aren't public, so I need to copy details here. It's failing everywhere because of a compile error:

> ##[error]src\NuGet.Clients\NuGet.CommandLine\Common\DependencyBehaviorHelper.cs(20,27): Error CS0246: The type or namespace name 'CommandLineException' could not be found (are you missing a using directive or an assembly reference?)

> ##[error]src\NuGet.Clients\NuGet.CommandLine\Commands\UpdateCommand.cs(288,89): Error CS0103: The name 'DependencyBehavior' does not exist in the current context

--- a/src/NuGet.Clients/NuGet.CommandLine/Commands/UpdateCommand.cs
+++ b/src/NuGet.Clients/NuGet.CommandLine/Commands/UpdateCommand.cs
@@ -21,6 +21,7 @@ using NuGet.Versioning;
 using NuGet.Packaging.Signing;
 using NuGet.Packaging;
 using NuGet.Packaging.PackageExtraction;
+using NuGet.Resolver;
 
 namespace NuGet.CommandLine
 {

--- a/src/NuGet.Clients/NuGet.CommandLine/Common/DependencyBehaviorHelper.cs
+++ b/src/NuGet.Clients/NuGet.CommandLine/Common/DependencyBehaviorHelper.cs
@@ -3,6 +3,7 @@
 
 using System;
 using System.Globalization;
+using NuGet.Commands;
 using NuGet.Configuration;
 using NuGet.Resolver;
 
@@ -17,7 +18,7 @@ namespace NuGet.CommandLine
             if (!Enum.TryParse<DependencyBehavior>(behaviorStr, ignoreCase: true, result: out dependencyBehavior) ||
                 !Enum.IsDefined(typeof(DependencyBehavior), dependencyBehavior))
             {
-                throw new CommandLineException(string.Format(CultureInfo.CurrentCulture,
+                throw new CommandException(string.Format(CultureInfo.CurrentCulture,
                     LocalizedResourceManager.GetString("Error_UnknownDependencyVersion"), behaviorStr));
             }
 

